openshift/openshift_transport.go

```diff
@@ -44,67 +43,33 @@ func (t openshiftTransport) ValidatePolicyConfigurationScope(scope string) error
 
 // openshiftReference is an ImageReference for OpenShift images.
 type openshiftReference struct {
-	baseURL         *url.URL
 	namespace       string
 	stream          string
-	tag             string
-	dockerReference reference.Named // Computed from the above in advance, so that later references can not fail.
+	dockerReference reference.NamedTagged // Computed from the above in advance, so that later references can not fail.
```

Comment:
This comment is obsolete—in fact it would apply to `namespace` and `stream` now.